ansible/templates/debian/deps.sh.j2

```diff
@@ -4,6 +4,9 @@ Install other dependencies. The number of dependencies to install depends on the
 #!/bin/bash
 
 set -e -u -x
+echo 'APT::Install-Recommends 0;' >> /etc/apt/apt.conf.d/01norecommends \
```

Comment:
Use `>` rather than `>>` to clear the file?  Otherwise the script won't be idempotent?

Reply:
On the other hand, if it's run on a user's system it could result in dataloss.  But maybe if it's only an example it's OK.

Reply:
not visible in the doc, just used to build the container. I will fix it